src/app/components/layout-sidebar/layout-sidebar-demo.component.html

```diff
@@ -1,3 +1,3 @@
-<stache-layout-sidebar>
+<stache layout="sidebar">
```

Comment:
I think we should pick one way to do it and then document that. Even having those two options is confusing.